test/test_priorityqueue.jl

```diff
@@ -93,6 +93,18 @@ while !isempty(pq)
     @test 10 != dequeue!(pq)
 end
 
+priorities2 = Dict(zip('a':'e', 5:-1:1))
+pq = PriorityQueue(priorities2)
+try
+    dequeue_pair!(pq, 'g')
+    error("should have resulted in KeyError")
```

Comment:
Better as `throw ErrorException(...)`. We're trying to remove uses of `error` in lieu of throwing errors.